jest.config.js

```diff
@@ -0,0 +1,11 @@
+// configurations for jest unit testing
+
+module.exports = {
+  preset: "@vue/cli-plugin-unit-jest",
+  transform: {
+    "^.+\\.vue$": "vue-jest",
+  },
```

Comment:
can you explain this a bit?

Reply:
I initially used a lot of articles to get the test cases working. I think this summarizes well.
https://vue-test-utils.vuejs.org/installation/#using-vue-test-utils-with-jest-recommended

jest.config.json got created automatically after running `vue add unit-jest`